argyrogounari/assignment1/Question2.java

```diff
@@ -0,0 +1,16 @@
+
+public class Question2 {
```

Comment:
Again, adding a javadoc statement indicating what the question is asking for would help.